compile.sh

```diff
@@ -37,7 +36,26 @@ if [ "$_has_dtbo" = "true" ]; then
 fi
 
 _self_dir=$(realpath $(dirname "$0"))
-. $_self_dir/compile_$_compiler.sh
+. $_self_dir/setup_$_compiler.sh
+
+_make_args+=" O=$_out ARCH=arm64 -j$(nproc)"
```

Comment:
I'd prefer to initialize `_make_args` with these common values before calling `setup_$_compiler.sh` and do the `+=` dance  there, but it's just a matter of personal taste and wouldn't mind doing this your way :)